ibis/backends/dask/udf.py

```diff
@@ -165,9 +165,10 @@ def lazy_agg(*series: pandas.Series):
         return result
 
     @execute_node.register(
-        type(op), *(itertools.repeat(ddgb.SeriesGroupBy, nargs))
+        ops.ReductionVectorizedUDF,
```

Comment:
I don't see tests changes around reductions. Are these changes covered by tests?

Reply:
None of the reduction codepaths changed, I just split the grouped analytic one to a new handler.